docs/platform/release-notes/tizen-7-0-m2.md

```diff
@@ -0,0 +1,543 @@
+    - Deprecated method of AudioManager has been removed.
+    - Tizen.WebView has been deprecated.
+    - EncodedPlane has been added to Tizen.Multimedia Camera.
+    - Camera and SceneView class has been added to Tizen.NUI.Scene3D.
+    - Privilege level of MediaScreenSource class has been changed from public to platform.
+
+
+### Toolchain
+
+#### New and changed features
+
+- GCC
+  - Timezone data has been updated to tzcode 2020a.
+
+#### Fixes
+
+- GCC
+  - C++ bugfix patches have been applied (PR c++/61414).
+  - Memcmp bugfix patches have been applied (PR middle-end/95189, 95886).
+- Glibc
+  - TLS(Thread Local Storage) bugfix patches have been applied (BZ #25051).
+
+
+### Machine learning
+
+#### New and changed features
+
+- Machine Learning (ML) API updates
+  - ML.Service API Updates
+    - Home Edge-AI and Among-Device AI support: applications may register and deploy AI service modules, which can be accessed by other applications in the device or connected devices. Such edge-AI or among-device AI capability is provided by the API set to launch / start / stop / destroy the AI pipeline descriptions, which has been newly added. Developers may provide the specific inference operations as ML service and other applications may use this service via the network. Note that this capability is compatible with SmartThings and Matter via AITT module and devices without nnstreamer (e.g., TizenRT) can be potentially connected by adopting nnstreamer-edge library.
+    - API set to create / request the query service to the ML service instance has been newly added. Applications may create the query for inference offloading and request it to the ML service instance with its data.
+  - ML.Inference API Updates
+    - API set to set / destroy the ML options has been newly added. Developers may use this API to set the various option of the deep neural network framework.
+    - Version 2.8.1 of TensorFlow Lite has been supported as default.
+  - ML.Training API Updates
+    - Added New Enumerations for layers
+      - ML_TRAIN_LAYER_TYPE_ATTENTION
+      - ML_TRAIN_LAYER_TYPE_MOL_ATTENTION
+      - ML_TRAIN_LAYER_TYPE_LAYER_NORMALIZATION
+      - ML_TRAIN_LAYER_TYPE_POSITIONAL_ENCODING
+    - .NET API has been added. Now, It is possible to construct, control, and train a machine learning model in Tizen device with Tizen.ManchineLearning.Trainng .NET Class.
+      - It provides :
+        - Tizen.MachineLeanring.Train.Model
+        - Tizen.MachineLearning.Train.Layer
+        - Tizen.MachineLearning.Train.Optimizer
+        - Tizen.MachineLearning.Train.Dataset
+        - And more.
+    - Training API in WebAPIs has been added for creation and training new models for machine learning on Tizen.
+      - It contains:
+        - Layer creattionLayer(LayterType type);
+        - Dataset createFileDataset()
+        - Layer createOptimizer()
+        - Model createModelWithConfiguration()
+        - Model createMode()

```

Comment:
Line 492 ends with a semi-colon `;`. Should the following lines also end with semi-colon `;`
```suggestion
        - Layer creattionLayer(LayterType type);
        - Dataset createFileDataset()
        - Layer createOptimizer()
        - Model createModelWithConfiguration()
        - Model createMode()

```

Reply:
This needs to be ```creationLayer(LayterType type)```